x/concentrated-liquidity/swaps_test.go

```diff
@@ -1602,38 +1602,6 @@ func (s *KeeperTestSuite) TestComputeAndSwapOutAmtGivenIn() {
 	}
 }
 
-func (s *KeeperTestSuite) TestSwapOutAmtGivenIn_NoPositions() {
-	s.SetupTest()
-
-	pool := s.PrepareConcentratedPool()
-
-	// perform swap
-	_, _, _, _, _, err := s.App.ConcentratedLiquidityKeeper.SwapOutAmtGivenIn(
-		s.Ctx, s.TestAccs[0], pool,
-		DefaultCoin0, DefaultCoin1.Denom,
-		sdk.ZeroDec(), sdk.ZeroDec(),
-	)
-
-	s.Require().Error(err)
-	s.Require().ErrorIs(err, types.NoSpotPriceWhenNoLiquidityError{PoolId: pool.GetId()})
-}
-
-func (s *KeeperTestSuite) TestSwapInAmtGivenOut_NoPositions() {
-	s.SetupTest()
-
-	pool := s.PrepareConcentratedPool()
-
-	// perform swap
-	_, _, _, _, _, err := s.App.ConcentratedLiquidityKeeper.SwapInAmtGivenOut(
-		s.Ctx, s.TestAccs[0], pool,
-		DefaultCoin0, DefaultCoin1.Denom,
-		sdk.ZeroDec(), sdk.ZeroDec(),
-	)
-
-	s.Require().Error(err)
-	s.Require().ErrorIs(err, types.NoSpotPriceWhenNoLiquidityError{PoolId: pool.GetId()})
-}
-
```

Comment:
why removing tests?

Reply:
becasue you're a good reviewer (I will restore now, this is def an error, thank you)